src/components/fundamentals/icons/cart-icon/cart-icon.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import { ComponentMeta } from "@storybook/react"
+import CartIcon from "."
+
+export default {
+  title: "Fundamentals/Icons/CartIcon",
+  component: CartIcon,
+  argTypes: {
+    size: {
+      control: {
+        type: "select",
+        options: ["24", "20", "16"],
+      },
+    },
+  },
+} as ComponentMeta<typeof CartIcon>
```

Comment:
suggestion: `const cartIconComponentMeta: ComponentMeta<typeof CartIcon> = {...}; export default cartIconComponentMeta` something like this should give it stricter typing.

Reply:
👍